test/utilities.py

```diff
@@ -6,7 +6,7 @@
 from urllib.parse import parse_qs, urlparse
 
 LOCAL_WEBSERVER_PORT = 8000
-BASE_TEST_URL_DOMAIN = "localtest.me"
+BASE_TEST_URL_DOMAIN = "0.0.0.0"
```

Comment:
Do we have a reason for using `0.0.0.0` instead of `127.0.0.1`/`localhost`?

Reply:
No, I didn't and I'm trying out localhost as I can't get the worker script  tests to pass.